flink-formats/flink-orc-nohive/pom.xml

```diff
@@ -90,6 +90,37 @@ under the License.
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.hadoop</groupId>
+			<artifactId>hadoop-common</artifactId>
+			<scope>provided</scope>
+			<exclusions>
+				<exclusion>
+					<groupId>ch.qos.reload4j</groupId>
+					<artifactId>reload4j</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.slf4j</groupId>
+					<artifactId>slf4j-reload4j</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
Thats because we dont allow Reload4J dependencies due to their conflict with Log4j -- we use maven-enforcer [rules] for that (https://github.com/apache/flink/blob/85efa13e5d16f7962113d9b905efcceb9b99f8a3/pom.xml#L1773):

Failed run example:
```
[ERROR] Rule 0: org.apache.maven.plugins.enforcer.BannedDependencies failed with message:
Log4j 1 and Reload4J dependencies are not allowed because they conflict with Log4j 2. If the dependency absolutely requires the Log4j 1 API, use 'org.apache.logging.log4j:log4j-1.2-api'.
Found Banned Dependency: ch.qos.reload4j:reload4j:jar:1.2.18.3
Found Banned Dependency: org.slf4j:slf4j-reload4j:jar:1.7.36
```